packages/core/src/editor/Block.css

```diff
@@ -372,20 +373,19 @@ NESTED BLOCKS
   height: 24px;
 }
 
-[data-file-block] .bn-visual-media-wrapper {
+[data-file-block] .bn-resize-handles-wrapper {
```

Comment:
make sure DO etc isn't using the old classnames. I suppose you didn't like the earlier name / just a cleanup right?